fe/fe-core/src/main/java/com/starrocks/alter/AlterJobMgr.java

```diff
@@ -475,6 +475,10 @@ public void processAlterTable(AlterTableStmt stmt) throws UserException {
                 // if modify storage type to v2, do schema change to convert all related tablets to segment v2 format
                 schemaChangeHandler.process(alterClauses, db, olapTable);
                 isSynchronous = false;
+            } else if (currentAlterOps.contains(AlterOpType.MODIFY_TABLE_PROPERTY_SYNC) &&
```

Comment:
The AlterOpType.MODIFY_TABLE_PROPERTY_SYNC here indicates this is a synchronous operation, but actually this is an asynchronous operation, right?

Reply:
Yes, because the sql syntax has not changed, maybe the `AlterOpType` can be determined based on `RUN_MODE`